duckdns/data/run.sh

```diff
@@ -65,8 +65,8 @@ fi
 # Run duckdns
 while true; do
 
-    [[ ${IPV4} != *:/* ]] && ipv4=${IPV4} || ipv4=$(curl -s -m 10 "${IPV4}")
-    [[ ${IPV6} != *:/* ]] && ipv6=${IPV6} || ipv6=$(curl -s -m 10 "${IPV6}")
+    [[ ${IPV4} != *:/* ]] && ipv4=${IPV4} || ipv4=$(curl -s -m 10 "${IPV4}") || bashio::log.warning "Couldn't retrieve ipv4 from server"
+    [[ ${IPV6} != *:/* ]] && ipv6=${IPV6} || ipv6=$(curl -s -m 10 "${IPV6}") || bashio::log.warning "Couldn't retrieve ipv6 from server"
 
     # Get IPv6-address from host interface
```

Comment:
From a security perspective, it would be nicer if the add-on does not require host network.

Is the IPv6 address retrial from the host interface in the following lines not working for you?

Reply:
No, unfortunately, `curl api6.ipify.org` cannot connect to the ipv6 server without the host network. 
In that case, only `curl api.ipify.org` works for the ipv4 address.

Reply:
Yes I get that. And I guess this line should be deleted, as it can't work.

However, following this code, there is a section which gets the IP address via API from the Supervisor:
https://github.com/home-assistant/addons/blob/master/duckdns/data/run.sh#L71-L82

Is this code not working for you?

Reply:
I see. Thank you. Yes, It works now when I set ipv6 to `default` in the configuration. Maybe you can remove the information about the URL and IPv6 from the documentation: 
`If you specify a URL here, contents of the resource it points to will be fetched and used as the address. This enables getting the address using a service like https://api6.ipify.org/ or https://ipv6.text.wtfismyip.com/` 

Moreover, it would be helpful to get any warnings in the logs as to why the addon is not working. I updated the branch with more detailed warnings. Furthermore, I changed that the addon uses the default interface automatically when nothing is specified. Maybe it makes sense to include these changes.

Reply:
Oh yes, I see, this documentation is really wrong :see_no_evil: 

Thanks for the updated PR! 

Probably a new PR would have been easier here :sweat_smile:  But if you stick with this one, can you also update the title and description?

Reply:
Thanks. I incorporated the changes you suggested and opened a new pull request #3460